lib/cartopy/mpl/geoaxes.py

```diff
@@ -1737,6 +1754,14 @@ def _pcolormesh_patched(self, *args, **kwargs):
                         # this method
                         collection._wrapped_collection_fix = pcolor_col
 
+                        # if t is self.projection and
+                        # there are overlapping cells
+                        # then pcolor won't work for those cells
+                        # so we need to hide them
+                        if t is self.projection:
```

Comment:
I don't think this is right. If this were the case, you'd also want to put this up above so that `pcolor` isn't ever called at all.

Reply:
The reason to keep it there and not above is to allow big cells. Let's say that we change the value, at your 2 comments above, to 1/3, then large cells will be never displayed. Here we produce them and hide them, so that the user (if we provide the right documentation) can still decide to make them visible. It means that in some case we may hide actual big cells not overlapping cells, and the user can make them visible again.